process/elasticproc/factory/elasticProcessorFactory.go

```diff
@@ -19,6 +19,14 @@ import (
 	"github.com/multiversx/mx-chain-es-indexer-go/process/elasticproc/validators"
 )
 
+// ElasticConfig holds the elastic search settings
+type ElasticConfig struct {
```

Comment:
Isn't this duplicated?

Reply:
Kind of, but the one from `config.go` cannot be used..